sunpy/extern/parse.py

```diff
@@ -169,7 +169,7 @@ def __eq__(self, other):
 DAYS_PAT = r"(Mon|Tue|Wed|Thu|Fri|Sat|Sun)"
```

Comment:
This file should not be touched, all the changes here need to be reverted.

Reply:
OK, will add tests and change as per the instructions.